src/metpy/plots/declarative.py

```diff
@@ -498,6 +500,23 @@ def lookup_map_feature(feature_name):
     return feat.with_scale(scaler)
 
 
+@exporter.export
+class SPCHatch(matplotlib.hatch.Shapes):
```

Comment:
Can we wait on the custom hatch? Matplotlib has deprecated passing in hatch values it doesn't understand in 3.4, with plans to remove it in 3.6. Thus using this would result in an error trying to use this when matplotlib 3.6 comes out. matplotlib/matplotlib#20690 is discussing having a proper API for adding custom hatches, so it would be good for you to weigh in there (maybe with the example of what yours looks like). You could also push that forward with a draft API if you're feeling particularly driven. :wink: